openassessment/assessment/models/staff.py

```diff
@@ -80,58 +83,35 @@ def get_workflow_statistics(cls, course_id, item_id):
         return {'ungraded': ungraded, 'in-progress': in_progress, 'graded': graded}
 
     @classmethod
-    def get_submission_for_review(cls, course_id, item_id, scorer_id):
+    def get_submission_for_review(cls, course_id, item_id):
         """
-        Find a submission for staff assessment. This function will find the next
-        submission that requires assessment, excluding any submission that has been
-        completely graded, or is actively being reviewed by other staff members.
+        Find all submissions for staff assessment.
 
         Args:
-            submission_uuid (str): The submission UUID from the student
+            course_id (str): The submission UUID from the student
```

Comment:
Is course_id is submission uuid? It little bit confused.

Reply:
Done